docs/linux/dm.md

```diff
@@ -291,30 +290,31 @@ have the same size as the zones of the backing zoned devices. A logical chunk
-   figure which is mapped to the conventional zone *C<sub>A</sub>*. This is the
-   default mapping initialized when the first write command is issued to an
-   empty (unwritten) chunk. As long as a chunk is mapped to a conventional zone,
-   any incoming write request can be directly executed using the mapped
-   conventional zone.
-2. **Sequential zone mapping** A chunk initially can also be mapped to a
-   sequential zone as shown for the chunk *C* mapped to the sequential zone
-   *S<sub>C</sub>* in the figure. With such mapping, a already written block of
-   the chunk cannot be modified directly. To handle this case, the next mapping
-   type is used.
-3. **Dual conventional-sequential zone mapping** To process data updates to
-   written blocks of a chunk mapped to a sequential zone, a conventional zone
-   may be temporarily added to the chunk mapping. Any write targeting a written
-   block will be processed using the conventional zone rather than the
-   sequential zone.
+   figure, which is mapped to the conventional zone *C<sub>A</sub>*. This is the
+   default mapping that is initialized when the first write command is issued 
+   to an empty (unwritten) chunk. As long as a chunk is mapped to a 
+   conventional zone, any incoming write request can be directly executed 
+   using the mapped conventional zone.
+2. **Sequential zone mapping** A chunk can be mapped initially to a
+   sequential zone, as shown for the chunk *C* (mapped to the sequential zone
+   *S<sub>C</sub>* in the figure). With such a mapping, an already-written 
+   block of the chunk cannot be modified directly. To handle this case, the 
+   next mapping type is used.
+3. **Dual conventional-sequential zone mapping** Temporarily add a conventional
+   zone to the chunk mapping when you need to update data in an already-written
+   block of a chunk that has been mapped to a sequential zone. Any write 
+   that targets a written block will be processed using the conventional 
+   zone instead of the sequential zone.
 
 *dm-zoned* metadata include a set of bitmaps to track the validity state of
-blocks in the backing device zones. Any write operation execution is always
-followed by an update to the bitmaps to mark the written blocks as valid. In the
-case of the dual conventional-sequential chunk mapping, the bitmap for the
-blocks of the sequential zone is also updated to clear the bits representing the
-blocks updated but written to the conventional zone. Doing so, incoming reads
-always gain access to the latest version of the block data by simply inspecting
-the block validity bitmaps.
+blocks in the zones of the backing device. Any write-operation execution is
+always followed by an update to the bitmaps, to mark the written blocks as
+valid. In the case of the dual conventional-sequential chunk mapping, the
+bitmap for the blocks of the sequential zone is updated to clear the bits
+that represent the blocks that have been updated, but it is written to the
```

Comment:
That change is weird, and the original sentence was not super nice either.
What about:

..., the bitmap for the blocks of the sequential zone is updated to clear the bits
that represent the blocks that have been updated with a write in the conventional zone.

Reply:
This is exactly the sentence that I referred to in the commit message when I said that some of this would require technical correction. I'm encouraged to see that we struck on the same sentence.